File: Molinos.Scato.Dominio/Entidades/PlanoDeCargaBodega.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Molinos.Scato.Dominio.Entidades
{
    public class PlanoDeCargaBodega : IIdentificable
    {
        [Key]
        public virtual int Id { get; set; }
        public virtual int BodegaParcel { get; set; }
        public virtual int Cantidad { get; set; }
        public virtual MaterialPuerto MaterialPuerto { get; set; }
        public virtual string Condicion { get; set; }
        public virtual string SfFull { get; set; }
        public virtual Destino Destino { get; set; }
        public virtual PlanoDeCarga PlanoDeCarga { get; set; }
        public virtual string TanqueDeAbordo { get; set; }
    }
}

File: Molinos.Scato.WebPuerto/src/app/shared/modelos/plano-de-carga-bodega.ts
import { Destino } from "./destino";
import { MaterialPuerto } from "./material-puerto";
import { PlanoDeCargaBodegaDestino } from "./plano-de-carga-bodega-destino";

export class PlanoDeCargaBodega {
      id : number;
      bodegaParcel: number;
      cantidad: number;
      materialPuerto: MaterialPuerto;
      condicion: string;
      sfFull: string;
      destino: Destino;
      tanqueDeAbordo: string;
      destinos: PlanoDeCargaBodegaDestino[];
      destinosPaises?: Destino[];
      mostrar?: boolean;
      public constructor(init?:Partial<PlanoDeCargaBodega>) {
            Object.assign(this, init);
      }
}
